### DISCUSSION TITLE
Items could be lower than 0

### DISCUSSION THREAD

**Discussion by AndresMorelos** [AndresMorelos/InvoncifyReleases#60] [Q&A] [answered]
```
It also would be nice if the price of an item could be lower than 0
(ex.deposit).
Kind regards
```

Ref:
https://github.com/AndresMorelos/Invoncify/discussions/31#discussioncomment-2582435

**Comment by AndresMorelos** [comment]
@bdlanduyt there is an in-built discount field you can use for this type of item.

**Comment by AndresMorelos** [comment, marked as answer]
@bdlanduyt The prepayments feature was implemented on the v1.15.0

**Comment by bdlanduyt** [comment]
Please review my current translations.

Op wo 11 mei 2022 om 04:15 schreef Andrés Morelos ***@***.***
